components/Palettes/src/MDCPalettes.m

```diff
@@ -42,6 +43,34 @@
                          alpha:1];
 }
 
+static void MDCPalettizeColors(MDCPalette *palette) {
```

Comment:
Naming is hard. But I can't think of a better one.

Reply:
Since it's static, we can drop "MDC". PreparePaletteColors(...)?

Reply:
Ah, I like "LinkPaletteColors" even better!

Reply:
Yes!